frontend/src/e2e-test/pages/citizen/citizen-messages.ts

```diff
@@ -148,16 +151,30 @@ export default class CitizenMessagesPage {
   async assertReplyContentIsEmpty() {
     return this.#messageReplyContent.assertTextEquals('')
   }
-
   async replyToFirstThread(content: string) {
+    await this.startReplyToFirstThread()
+    await this.#messageReplyContent.fill(content)
+    await this.sendReply()
+  }
+  async startReplyToFirstThread() {
     await this.#threadListItem.click()
     await this.#openReplyEditorButton.click()
-    await this.#messageReplyContent.fill(content)
+  }
+  getReplyContentElement() {
```

Comment:
Yksinkertaisempi ratkaisu olisi vaan paljastaa `messageReplyContent` suoraan ulospäin (`#messageReplyContent: Element` -> `messageReplyContent: Element`). Nämä yhden rivin wrapperit on 99.9% ajasta turhia, varsinkin kun tässä käytännössä vaan vaihdetaan yksi nimi ("message reply content") toiseen ("reply content element")

Reply:
Korjattu